### PR TITLE
Include net45 framework references in all packages

### PR DESCRIPTION
**Pull request by mellinoe**
This is the counterpart to corefx's fix: https://github.com/dotnet/corefx/pull/8791.

Currently, if you reference these packages from a net45 project, you will not get any reference assemblies added. The types for these projects are all implemented in System.ServiceModel.dll, but you need to manually add that reference, even after referencing the nuget package. This change will cause people who install the package to automatically receive a reference to System.ServiceModel.dll. This already happens for people who install the packages into a net46 project, because we are able to automatically harvest the appropriate references from the live-build facade assembly.

This also updates buildtools to version 1.0.25-prerelease-00424-01, which contains the change in Packaging.targets which understands this new metadata item.

@ericstj

### CODE REVIEW

**Comment by roncain** [comment]
@dotnet-bot test outerloop Windows_NT
@dotnet-bot test outerloop selfhost Windows_NT

**Comment by StephenBonikowsky** [comment]
LGTM, pending CI checks.

**Comment by roncain** [comment]
@dotnet-bot test outerloop Ubuntu14.04

File: BuildToolsVersion.txt [hunk 1]
```diff
@@ -1 +1 @@
-1.0.25-prerelease-00420-01
\ No newline at end of file
+1.0.25-prerelease-00424-01
```
Comment (roncain) [inline BuildToolsVersion.txt]: I notice CoreFx is now at 00425-03.  @mellinoe -- do you mind updating to match, or is there a reason not to?

Comment (roncain) [inline BuildToolsVersion.txt]: @mellinoe -- we've decided to merge yours as is so PR #1214 can follow yours and update, so no need to update your PR.

**Pull request merged**